### DISCUSSION TITLE
Is there a `bottomk` query?

### DISCUSSION THREAD

**Discussion by clippered** [milvus-io/milvus#4464] [Q&A and General discussion]
From the query DSL, we only have `topk` which gets the top most values. Is there a, maybe something like `bottomk` , which gets the bottom most values?
Thanks

**Comment by shiyu22** [comment]
Hi,
currently only `topk` is supported, if you need `bottomk`, please submit a [feature issue](https://github.com/milvus-io/milvus/issues/new/choose) and introduce your requirement scenario to us.
